### PR TITLE
Stable API | Remove shipping simulator from *-theme-vtex

### PR DESCRIPTION
**Pull request by tlgimenes**
## What's the purpose of this pull request?
<em>Considering the context, what is the problem we'll solve? Where in VTEX's big picture our issue fits in? Write a tweet about the context and the problem itself.</em>

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?
<em>Describe the steps with bullet points. Is there any external link that can be used to better test it or an example?</em> 

## References
<em>Spread the knowledge: is there any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: adding references to related issues or mentioning people important to this PR may be good for the documentation and reviewing process</em>

### CODE REVIEW

File: packages/gatsby-theme-store/src/components/ShippingSimulator/index.tsx [hunk 1]
```diff
@@ -1,59 +0,0 @@
-import React from 'react'
```
Comment (klzns) [inline packages/gatsby-theme-store/src/components/ShippingSimulator/index.tsx:1]: Why removed this component?

Comment (Gmantiqueira) [reply packages/gatsby-theme-store/src/components/ShippingSimulator/index.tsx:1]: It looks like it was only used in Marin Brasil and Gimenes made [some changes there](https://github.com/vtex-sites/marinbrasil.store/pull/439)

**Review by Gmantiqueira — APPROVED** [review]
LGTM!

File: packages/gatsby-theme-store/src/components/ShippingSimulator/index.tsx (hunk 1 above)
Comment (tlgimenes) [reply packages/gatsby-theme-store/src/components/ShippingSimulator/index.tsx:1]: The idea is to have the glue code between sdk and ui in the store's own code

**Pull request merged**